File: beginner_programming/age_counter/java/CountAgeOfPerson.java
import java.time.LocalDate;
import java.time.Period;
import java.util.Scanner;

/**
 * Java implementation of counting a person's age in days
 */
public class CountAgeOfPerson {

    /**
     * Main method
     */
    public static void main(String[] args) {

        //get user's birthday
        int[] birthdayArray = getUserBirthday();
        
        //convert birthday into LocalDate
        LocalDate birthday = LocalDate.of(birthdayArray[0], birthdayArray[1], birthdayArray[2]);

        //get today's date
        LocalDate today = LocalDate.now();
        
        //calculate period between today's date and the user's birthday
        Period p = Period.between(birthday, today);

        //print out the final message
        System.out.println("You are " + p.getYears() + " years, " + p.getMonths() + " months and " + p.getDays() + " days old.");
    }

    /**
     * Method which obtains the users birthday as a string
     */
    public static int[] getUserBirthday() {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter your birth year: ");
        int year = sc.nextInt();
        System.out.print("Enter your birth month: ");
        int month = sc.nextInt();
        System.out.print("Enter your birth day: ");
        int day = sc.nextInt();
        sc.close();
        return new int[]{year, month, day};
    }
}

File: beginner_programming/age_counter/c/count_age_of_person.c
#include<stdio.h>

int main()
{
	int BirthMonth, BirthYear;
	int CurrentMonth, CurrentYear;
	int AgeYear, AgeMonth;
	
	printf("\n\n\t\t\tCount the age person\n\n");
	printf("Enter Your Birth Year(Eg:1989):");
	scanf("%d",&BirthYear);
	
	printf("\n\nEnter Your Birth Month(Eg:7):");
	scanf("%d",&BirthMonth);
	
	printf("\nEnter The Current Month(Eg:7):");
	scanf("%d",&CurrentMonth);
	
	printf("\nEnter The Current Year(Eg:2010):");
	scanf("%d",&CurrentYear);
	
	AgeYear=CurrentYear-BirthYear;
	AgeMonth=CurrentMonth-BirthMonth;
	
	printf("\n\n\t\tYour Age is %d Years And %d Months ",AgeYear,AgeMonth);
	
	return 0;
}
